doc/releasenotes/16_0_0_summary.md

```diff
@@ -177,18 +152,43 @@ The stats `QueriesProcessed` and `QueriesRouted` are deprecated in v16. The same
 
-During upgrades, if the users want to preserve the same behaviour as previous releases, then they should provide the `remote_operation_timeout` flag explicitly before upgrading.
-After the upgrade, they should then alter their configuration to also specify `lock-timeout` explicitly.
+### <a id="repl-manager-removal"/>Replication manager removal and VTOrc becomes mandatory
+VTOrc is now a **required** component of Vitess starting from v16. If the users want VTOrc to manage replication, then they must run VTOrc.
+Replication manager is removed from vttablets since the responsibility of fixing replication lies entirely with VTOrc now.
+The flag `disable-replication-manager` is deprecated and will be removed in a later release.
 
-#### <a id="normalized-lables"/>Normalized labels in the Prometheus Exporter
+### <a id="vreplication"/>VReplication
 
-The Prometheus metrics exporter now properly normalizes _all_ label names into their `snake_case` form, as it is idiomatic for Prometheus metrics. Previously, Vitess instances were emitting inconsistent labels for their metrics, with some of them being `CamelCase` and others being `snake_case`.
+#### <a id="vstream-copy-resume"/>VStream Copy Resume
+
+In [PR #11103](https://github.com/vitessio/vitess/pull/11103) we introduced the ability to resume a `VTGate` [`VStream` copy operation](https://vitess.io/docs/design-docs/vreplication/vstream/vscopy/). This is useful when a [`VStream` copy operation](https://vitess.io/docs/design-docs/vreplication/vstream/vscopy/) is interrupted due to e.g. a network failure or a server restart. The `VStream` copy operation can be resumed by specifying each table's last seen primary key value in the `VStream` request. Please see the [`VStream` docs](https://vitess.io/docs/16.0/reference/vreplication/vstream/) for more details.
+
+#### <a id="vdiff2-ga"/>VDiff2 GA
+
+We are marking [VDiff v2](https://vitess.io/docs/16.0/reference/vreplication/vdiff2/) as Generally Available or production-ready in v16. We now recommend that you use v2 rather than v1 going forward. V1 will be deprecated and eventually removed in future releases.
+If you wish to use v1 for any reason, you will now need to specify the `--v1` flag.
+
+### <a id="tablet-throttler"/>Tablet throttler
+
+The tablet throttler can now be configured dynamically. Configuration is now found in the topo service, and applies to all tablets in all shards and cells of a given keyspace. For backwards compatibility `v16` still supports `vttablet`-based command line flags for throttler ocnfiguration.
+
+It is possible to enable/disable, to change throttling threshold as well as the throttler query.
+
+See https://github.com/vitessio/vitess/pull/11604
+
+### <a id="inc-backup"/>Incremental backup and point in time recovery
+
+In [PR #11097](https://github.com/vitessio/vitess/pull/11097) we introduced native incremental backup and point in time recovery:
+
```

Comment:
Should we include PR links at all? If we do, it seems like we should do them for all the "major changes" in the release notes, or none of them. As it stands today, it is somewhat inconsistent.

Reply:
I have changed the summary to include links to PRs in most of the major changes.